openmc/region.py

```diff
@@ -46,6 +46,27 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
+    def update_surfaces(self, surfaces=None):
```

Comment:
I think this name is a little misleading -- it sounds like we are updating something about the surfaces. Can you change this and the other instances to `get_surfaces`?